packages/@aws-cdk/core/lib/bundling.ts

```diff
@@ -112,7 +112,7 @@ export class BundlingDockerImage {
       ...options.user
         ? ['-u', options.user]
         : [],
-      ...flatten(volumes.map(v => ['-v', `${v.hostPath}:${v.containerPath}`])),
+      ...flatten(volumes.map(v => ['-v', `${v.hostPath}:${v.containerPath}:delegated`])),
```

Comment:
We should model this as an additional `consistency` option on `DockerVolume` with a default of `delegated`.

Something like this:
```ts
export interface DockerVolume {
  /**
   * The path to the file or directory on the host machine
   */
  readonly hostPath: string;

  /**
   * The path where the file or directory is mounted in the container
   */
  readonly containerPath: string;

  /**
   * Mount consistency
   * 
   * @default DELEGATED
   */
  readonly consistency: DockerConsistency; // enum
}
```

Reply:
Looks good. Is that the terminology from docker?

Reply:
Yes, https://docs.docker.com/storage/bind-mounts/#configure-mount-consistency-for-macos

Reply:
Flag implemented.